src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -1,9 +1,22 @@
 <%@ page contentType="text/html;charset=UTF-8" language="java" %>
 <html>
 <head>
-    <title>Title</title>
+    <title>Taxi Service</title>
 </head>
 <body>
-<h1>Hello World!</h1>
+<h1>Welcome to Taxi Service!</h1>
+<p>
```

Comment:
tag "p" need to describe block with text, but if it just link without text you need not to add tag